src/Symfony/Component/AutoMapper/Extractor/PrivateReflectionExtractor.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AutoMapper\Extractor;
```

Comment:
Shouldn't this class be moved in the PropertyInfo component? At least it should be marked `@internal`.

Reply:
Yes it should,like the serializer normalizer bridge.